book/chapter-00.md

```diff
@@ -58,21 +58,21 @@ Example of the expected reward for each miner depending on which chain they work
 | Work on Majority Chain |          5, 5          |          -1, 7         |
 | Work on Minority Chain |          7, -1         |         -1, -1         |
 
-The Nash Equilibrium here is (5, 5)--both miners working on the Majority Chain.
+The Nash Equilibrium here is (5, 5) - both miners working on the Majority Chain.
 
-Since mining on the shorter fork does not provide an incentive, and there is a cost to building on a chain that is mitigated by a block reward, the result is a convergence around the longest chain. However, it is important to note that Bitcoin considers the defacto chain to be the one with the most cumulative proof of work rather than simply the longest chain.
+Since mining on the minority chain (that is, the shorter fork) does not provide an incentive, and there is a cost to building on a chain that is mitigated by a block reward, the result is a convergence around the longest chain. However, it is important to note that Bitcoin considers the defacto chain to be the one with the most cumulative proof of work rather than simply the longest chain.
```

Comment:
This line ("Since mining...longest chain") is kind-of redundant of the stuff in the paragraph right above "Example of the..." Especially if you accept my edits. Maybe delete it?